backend/Dockerfile

```diff
@@ -16,5 +16,17 @@ COPY --from=builder /app ./
 RUN apk upgrade --update-cache --available && \
     apk add openssl && \
     rm -rf /var/cache/apk/*
+
```

Comment:
It would be better to add comment on why we need to include chromium in the image, just like above comments `Dependency for Prisma`.

Reply:
Added annotations like Prisma. It looks like better to understand. Thanks.